scripts/monitor/notebook/plot-cromwell-bq-monitoring.ipynb

```diff
@@ -0,0 +1,1136 @@
+{
```

Comment:
<p>Take out this example if you want to use the actual example below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/broadinstitute/long-read-pipelines/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>The example below was removed and replaced with comments</p>